examples_deepspeed/universal_checkpointing/README.md

```diff
@@ -77,6 +77,40 @@ Please see the corresponding [pull request](https://github.com/microsoft/Megatro
 
 Combining sequence parallelism with data parallelism is another good use case for universal checkpointing, see [sp pull request](https://github.com/microsoft/DeepSpeed/pull/4752) for example and visualization of matching loss values.
 
+### TensorBoard Log Analysis
+
+The Universal Checkpointing example includes a TensorBoard analysis script that'll generate `csv` files and `png` plots across the unviersal checkpointing training steps for comparison of training and validation loss curves.
+
+The script can be executed as follows after Step 3 is completed:
```

Comment:
Maybe mention "can only" or "must" be executed after Step 3?

Reply:
Rephrased the wording to start with step 3 completion, followed by the command.